modules/Makefile.kubernetes

```diff
@@ -71,6 +72,15 @@ KUBECTL_CMD ?= $(KUBECTL_SSH_CMD) "$(KUBECTL)" --logtostderr=true --insecure-ski
 # Reference Docs:
 #  - https://cloud.google.com/container-engine/docs/kubectl/
 #
+## Bring up SSH tunnel for kubernetes commands, must be called before executing any targets
+kubernetes\:tunnel-up:
```

Comment:
You also can't auto apply to all targets since `-M` option to ssh command would force you to need to MFA again